dbtmetabase/parsers/dbt_manifest.py

```diff
@@ -31,6 +31,11 @@ def read_models(
         includes = self.includes
         excludes = self.excludes
 
+        if includes is not None:
```

Comment:
You don't need to assign it and then re-process, just replace the above assignments with 

```py
includes = [x.lower() for x in self.includes] if self.includes else []
excludes = [x.lower() for x in self.excludes] if self.excludes else []
```

Reply:
That looks much better, thanks! Done.